src/util.ts

```diff
@@ -792,8 +792,21 @@ export const isEIP1559Transaction = (transaction: Transaction): boolean => {
   );
 };
 
-export const convertHexToDecimal = (value: string | undefined): number =>
-  parseInt(value === undefined ? '0x0' : value, 16);
+/**
+ * Converts valid hex strings to decimal numbers, and handles unexpected arg types.
+ *
+ * @param value - a string that is either a hexadecimal with `0x` prefix or a decimal string.
+ * @returns a decimal number.
+ */
+export const convertHexToDecimal = (
```

Comment:
I modified this method to handle unexpected arg types, specifically decimal strings, rather than check if its a valid hex before passing it to the method each time. Not sure if it makes sense to make this light helper function more complex, but thought I'd see how others feel.